custom-fonts.md

```diff
@@ -139,3 +139,267 @@ object specifies a weight or style for which is there is no exact font file, the
+
+class _FontsPageState extends State<FontsPage> {
+  @override
+  Widget build(BuildContext context) {
+    // Roboto - Included with Material Design
+    var robotoContainer = new Container(
+      child: new Column(
+        children: <Widget>[
+          new Text(
+            "Roboto",
+          ),
+          new Text(
+            words1,
+            textAlign: TextAlign.center,
+            style: new TextStyle(
+              fontFamily: "Roboto",
+              fontSize: 20.0,
+            ),
+          ),
+        ],
+      ),
+      margin: const EdgeInsets.all(10.0),
+      padding: const EdgeInsets.all(10.0),
+      decoration: new BoxDecoration(
+        color: Colors.grey.shade200,
+        borderRadius: new BorderRadius.all(new Radius.circular(5.0)),
+      ),
+    );
+
+    // Rock Salt - https://fonts.google.com/specimen/Rock+Salt
+    var rockSaltContainer = new Container(
+      child: new Column(
+        children: <Widget>[
+          new Text(
+            "Rock Salt",
+          ),
+          new Text(
+            words2,
+            textAlign: TextAlign.center,
+            style: new TextStyle(
+              fontFamily: "Rock Salt",
+              fontSize: 17.0,
+            ),
+          ),
+        ],
+      ),
+      margin: const EdgeInsets.all(10.0),
+      padding: const EdgeInsets.all(10.0),
+      decoration: new BoxDecoration(
+        color: Colors.grey.shade200,
+        borderRadius: new BorderRadius.all(new Radius.circular(5.0)),
+      ),
+    );
+
+    // VT323 - https://fonts.google.com/specimen/VT323
+    var v2t323Container = new Container(
+      child: new Column(
+        children: <Widget>[
+          new Text(
+            "VT323",
+          ),
+          new Text(
+            words3,
+            textAlign: TextAlign.center,
+            style: new TextStyle(
+              fontFamily: "VT323",
+              fontSize: 25.0,
+            ),
+          ),
+        ],
+      ),
+      margin: const EdgeInsets.all(10.0),
+      padding: const EdgeInsets.all(10.0),
+      decoration: new BoxDecoration(
+        color: Colors.grey.shade200,
+        borderRadius: new BorderRadius.all(new Radius.circular(5.0)),
+      ),
+    );
+
+    // https://fonts.google.com/specimen/Ewert
+    var ewertContainer = new Container(
+      child: new Column(
+        children: <Widget>[
+          new Text(
+            "Ewert",
+          ),
+          new Text(
+            words4,
+            textAlign: TextAlign.center,
+            style: new TextStyle(
+              fontFamily: "Ewert",
+              fontSize: 25.0,
+            ),
+          ),
+        ],
+      ),
+      margin: const EdgeInsets.all(10.0),
+      padding: const EdgeInsets.all(10.0),
+      decoration: new BoxDecoration(
+        color: Colors.grey.shade200,
+        borderRadius: new BorderRadius.all(new Radius.circular(5.0)),
+      ),
+    );
+
+    // Material Icons font - included with Material Design
+    String icons = "";
```

Comment:
only if you want to. Making Travis happy and optimizing the images is the important part.